runtime/Cargo.toml

```
@@ -16,6 +16,9 @@ doctest = false
 anyhow = { workspace = true }
 async-trait = { workspace = true }
 chrono = { workspace = true }
+opentelemetry = { workspace = true }
+opentelemetry-proto = { workspace = true, features = ["gen-tonic", "logs"] }
```

```suggestion
opentelemetry-proto = { workspace = true, features = ["gen-tonic", "traces"] }
```
I might be wrong here :thinking:

---

... maybe most of these can be removed instead since we are using the tracing otel layer directly

---

We can try it out :+1:

---

We need the types from these deps for the shuttle_main codegen, so I don't think we can remove them. :thinking:

---

But we can make opentelemetry_proto a dev dependency, it's only used in the tests.